db/seeds.rb

```diff
@@ -0,0 +1,67 @@
+damien = Apartment.create(address: "6005 Damien Corners", monthly_rent: 400, sqft: 230, num_beds:4 , num_baths:1 )
+koch = Apartment.create(address: "95599 Koch Stream", monthly_rent: 2400, sqft: 1900, num_beds: 2, num_baths: 4)
+hal = Apartment.create(address: "40583 Hal Crossing", monthly_rent: 200, sqft: 800, num_beds: 3, num_baths: 2)
+verna = Apartment.create(address: "62897 Verna Walk", monthly_rent: 2400, sqft: 700, num_beds:2 , num_baths:3 )
+dibbert = Apartment.create(address: "351 Dibbert Fields", monthly_rent: 2300, sqft: 1300, num_beds: 2, num_baths: 1)
+buford = Apartment.create(address: "3710 Buford Passage", monthly_rent: 500, sqft: 700, num_beds: 1, num_baths: 4)
+tyree = Apartment.create(address: "64329 Tyree Creek", monthly_rent: 500, sqft: 2000, num_beds: 1, num_baths: 3)
+runte = Apartment.create(address: "17297 Runte Bypass", monthly_rent: 2300, sqft: 1700, num_beds: 3, num_baths: 2)
+marvin = Apartment.create(address: "0889 Marvin Radial", monthly_rent: 1100, sqft: 500, num_beds: 3, num_baths: 2)
+freddy = Apartment.create(address: "2745 Freddy Vista", monthly_rent: 2800, sqft: 1400, num_beds: 4, num_baths: 4)
+gutmann = Apartment.create(address: "359 Gutmann Pike", monthly_rent: 2700, sqft: 1900, num_beds: 2, num_baths: 1)
+howell = Apartment.create(address: "08465 Howell Harbor", monthly_rent: 300, sqft: 1300, num_beds:3 , num_baths:1 )
+borer = Apartment.create(address: "8865 Borer Viaduct", monthly_rent: 600, sqft: 600, num_beds: 1, num_baths: 4)
+emard = Apartment.create(address: "7357 Emard Row", monthly_rent: , sqft: 2300, num_beds: 3, num_baths: 2)
+benedict = Apartment.create(address: "175 Benedict Ridges", monthly_rent: 2600, sqft: 2100, num_beds: 3, num_baths: 3)
+schinner = Apartment.create(address: "45714 Schinner Expressway", monthly_rent: 900, sqft: 2200, num_beds: 6, num_baths: 3)
+leta = Apartment.create(address: "60059 Leta Roads", monthly_rent: 1100, sqft: 2500, num_beds: 3, num_baths: 4)
+
+leta.tenants.create(name: "Maudie Mosciski", age: 90, gender: "Female")
+clemenargaertine.tenants.create(name: "Kristin Wisoky", age: 23, gender: 'Female')
+lemke.tenants.create(name: "Demario King", age: 71, gender: 'Male')
+jany.tenants.create(name: "Kaitlin Cormier", age: 91, gender: 'Female')
+ivy.tenants.create(name: "Rosalinda Champlin", age: 39, gender: 'Female')
+kamille.tenants.create(name: "Valentin Keebler", age: 74, gender: 'Male')
+damien.tenants.create(name: "Ms. Piggy", age: 24, gender: "Female")
+koch.tenants.create(name: "Eryn Lynch", age: 12, gender: 'Female')
+hal.tenants.create(name: "Tiara Conn", age: 3, gender: 'Female')
+buford.tenants.create(name: "America Pollich", age: 82, gender: 'Female')
+tyree.tenants.create(name: "Mylene Krajcik", age: 25, gender: 'Female')
+runte.tenants.create(name: "Josh Gottlieb", age: 8, gender: 'Male')
+marvin.tenants.create(name: "Sherwood Stiedemann", age: 72, gender: 'Female')
+leta.tenants.create(name: "Katarina Kunde", age: 6, gender: 'Female')
+
```

Comment:
That's a lot of seed data.  Looks like you had fun.  I thought this might reference something fun, but I couldn't find a connection.